src/cdomains/apron/sharedDomain.apron.ml

```diff
@@ -0,0 +1,291 @@
+open Cil
```

Comment:
In what sense is `sharedDomain` a `domain` here? Maybe rename?

Reply:
I've renamed it to `sharedFunctions`. Although, I'm actually not sure if that's much better.